### PR TITLE
MAINTAINERS_GUIDE|CONTRIBUTING: Make generic enough for all OCI Projects

### PR DESCRIPTION
**Pull request by wking**
With opencontainers/tob#18 landing and runtime-tools and image-tools getting [seeded with project-template content](https://github.com/opencontainers/tob/pull/18/files#diff-07f935bfb8e1871445d8bf2ad3759f9bR40), it would be nice to make more of this repository generic enough to apply to all OCI Projects.  This PR addresses that (see also #4).

There are a number of commits here, and I've tried to keep them fairly atomic and independent.  If it makes review easier, I'm happy to split them out into separate PRs.  Detailed motivation for all changes is in the respective commit message.

This could be batched with #18 and #19 for an all-maintainers vote, but:
- It's touching very different stuff, so there's less maintainer-bandwidth savings by batching.
- It's a lot bigger, so it probably needs more editing before it settles enough to be worth voting on.

I recommend interested parties review this here, and once it settles down (likely long after #18 and #19 have been decided ;) we put it to a separate all-maintainer vote.

### CODE REVIEW

File: MAINTAINERS_GUIDE.md [hunk 1]
```diff
@@ -44,14 +41,19 @@ repository. An implementation change is a change to the source code. An
...
-this (see below "Who decides what?")
+1. Discuss a proposal on the [mailing list](CONTRIBUTING.md#mailing-list).
+   Anyone can do this.
+2. Open a pull request.
+   Anyone can do this.
+3. Discuss the pull request.
+   Anyone can do this.
+4. Endorse (`LGTM`) or oppose (`Rejected`) the pull request.
+   The relevant maintainers do this (see below [Who decides what?](#who-decides-what)).
+   Changes that effect project management (changing policy, cutting releases, etc.) are [proposed and voted on the mailing list](GOVERNANCE.md).
```
Comment (jonboulle) [inline MAINTAINERS_GUIDE.md]: affect

Comment (wking) [inline MAINTAINERS_GUIDE.md]: Thanks.  Fixed with 0787a41 → 9597f27.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -24,22 +49,18 @@ Fork the repo and make changes on your fork in a feature branch:
 - If it's a feature branch, create an enhancement issue to announce your
   intentions, and name it XXX-something where XXX is the number of the issue.
 
-Small changes or changes that have been discussed on the project mailing list
-may be submitted without a leader issue, in which case you are free to name
-your branch however you like.
+Small changes or changes that have been discussed on the
```
Comment (jonboulle) [inline CONTRIBUTING.md]: nit, this file is now inconsistently wrapped, grumble grumble

**Comment by jonboulle** [comment]
lgtm

File: CONTRIBUTING.md (hunk 2 above)
Comment (wking) [inline CONTRIBUTING.md]: I've been trying to transition to the one-sentence-per-line approach [used by runtime-spec][1] when I'd need to re-wrap anything.  I've updated this hunk to follow that with 9597f27 → d3f0172.  But the file as a whole still mixes manually wrapped lines (at 79 chars?) with the sentence wrapped lines I've touched.  Should I add a re-wrapping commit that applies sentence wrapping to everything else in the file?  Should I reroll my changes to stick with 79-char wrapping (and if so, should we document that wrapping width somewhere)?

[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc3/style.md#one-sentence-per-line

File: MAINTAINERS_GUIDE.md [hunk 3]
```diff
@@ -68,25 +70,10 @@ two LGTMs. In addition, if a maintainer has created a pull request, they cannot
 count toward the two LGTM rule (to ensure equal amounts of review for every pull
 request, no matter who wrote it).
 
-Overall the maintainer system works because of mutual respect across the
-maintainers of the project.  The maintainers trust one another to make decisions
-in the best interests of the project.  Sometimes maintainers can disagree and
-this is part of a healthy project to represent the point of views of various people.
-In the case where maintainers cannot find agreement on a specific change the
-role of a Chief Maintainer comes into play.
```
Comment (stevvooe) [inline MAINTAINERS_GUIDE.md]: Is the intent to remove the concept of a chief maintainer?

Comment (wking) [inline MAINTAINERS_GUIDE.md]: Yes.  See  the d3f0172 commit message for more motivation around this specific pivot.

Comment (stevvooe) [inline MAINTAINERS_GUIDE.md]: Which proposal removed the concept of Chief maintainer? That sounds like a necessary decider to keep things moving.

Comment (wking) [inline MAINTAINERS_GUIDE.md]: None of the changes in this PR have been voted on by any OCI maintainers.  As I said [in the topic post][1]

> I recommend interested parties review this here, and once it settles down (likely long after #18 and #19 have been decided ;) we put it to a separate all-maintainer vote.

If any portions of this PR are considered more contentious than others, I'm also happy to pull those commits off into a separate PR for a separate all-maintainer vote.

[1]: https://github.com/opencontainers/project-template/pull/20#issue-175942565

**Comment by wking** [comment]
Rebased around #27 with d3f0172 → 14a58e0.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,4 +1,30 @@
...
+
+The content of this repository is licensed under the [Apache License, Version 2.0](LICENSE).
+
+## Code of Conduct
+
+Participation in the Open Container community is governed by [Open Container Code of Conduct][code-of-conduct].
+
+## Weekly Call
+
+The contributors and maintainers of all OCI projects have a weekly meeting Wednesdays at 2:00 PM (USA Pacific).
```
Comment (RobDolinMS) [inline CONTRIBUTING.md]: Should this now be a monthly meeting?

Comment (wking) [inline CONTRIBUTING.md]: > Should this now be a monthly meeting?

Good point.  Done in 14a58e0 → dc9715d, although that means that new spec projects (if we get any) will need their own supplemental meetings to satisfy [this SHOULD][1].

[1]: https://github.com/opencontainers/project-template/blame/8e4c7e3dbea403c727d7fc526b8dcd53d6c98202/RELEASES.md#L34

**Comment by caniszczyk** [comment]
@wking if you can close this and break changes up and do it one PR a a time, that would be great, totally appreciate you cleaning this up

**Comment by wking** [comment]
Split up into #43 through #52, so closing this one.
